app/views/shared/_flashes.html.erb

```diff
@@ -0,0 +1,34 @@
+<% if [flash[:success], flash[:notice], flash[:multi_line_notice], flash[:warning], flash[:error]].any? %>
+  <div class="flashes">
+    <% if flash[:success] %>
+      <div class="alert alert-success alert-dismissible" role="alert">
+        <button type="button" class="btn-close" data-dismiss="alert" aria-label="Close"></button>
```

Comment:
access team used class "close" for the button.  It seemed less impactful to use the expected btn-close for alert-dismissable buttons and modify the scss slightly.

Reply:
WFM